content/sigstore-go-1-0-now-available.md

```diff
@@ -0,0 +1,18 @@
++++
+title = "sigstore-go 1.0 is now available"
+author = "Cody Soyland (GitHub)"
+date = "2025-05-12"
+tags = ["sigstore", "go", "clients"]
+type = "post"
+draft = false
++++
+
+We love Go within the Sigstore community, and it's been our language of choice since we got started. Cosign, Rekor, Fulcio, Policy Controller, and Timestamp Authority are all written in Go, and we're lucky to have such a vibrant community of Go developers.
+
+Cosign was the de-facto Sigstore "client" from the beginning. Originally designed as a container image signing tool, it has become much more, introducing "keyless signing" using Fulcio, blob signing, attestation support, multi-cloud KMS support, and many more features. Cosign was designed first and foremost as a CLI, and it continues to fulfill that role admirably. However, as we started to build more tools around Sigstore, the Go API started to show limitations, and the dependency tree grew larger. Additionally, as the community grew, we wanted to standardize around simpler data structures and specification-driven workflows.
```

Comment:
To add onto the last sentence, we could also mention client conformance?

Reply:
Yeah, mentioning the other clients and conformance tests is a good call!  I was just reading the [original announcement post](https://blog.sigstore.dev/announcing-sigstore-go/) again for inspiration and conformance tests were a big deal!

Reply:
Can we replace "keyless signing" to "signing with ephemeral keys"? There are still keys so the term "keyless" can be misleading.

I know it's been used frequently within the community but I'm doing my best to get away from it 🙂

Reply:
^Thank you! I've tried to do the same :)